src/app/authentication/register/register.component.scss

```diff
@@ -0,0 +1,78 @@
+.btn-custom {
+    border: 1px solid #ed1c24;
+    background-color: rgba(255, 255, 255, 0.6);
+    border-radius: 0px;
+    color: #333;
+    font-size: 15px;
+    font-family: 'Roboto', sans-serif;
+    font-weight: normal;
+    text-transform: uppercase;
+    outline: none;
+    padding: 4px 36px 2px;
+}
+
+.sign-in-page {
+    margin-top: 150px;
+    background-color: #fff;
+    border: 1px #e5e5e5 solid;
+    border-bottom: 2px #e5e5e5 solid;
+    padding: 20px;
+    overflow: hidden;
+}
+
+.move-left {
+    margin-left: 16.667%;
+    width: 66.667%;
+}
+
+.create-new-account h4 {
+    font-size: 18px;
+    font-family: 'Open Sans', sans-serif;
+    padding-bottom: 14px;
+    border-bottom: 1px solid #ddd;
+}
+
+.create-new-account p {
+    font-size: 15px;
+    color: #666;
+}
+
+.outer-top-xs {
+    margin-top: 20px;
+}
+
+.box {
+    display: inline-block;
+    border: 1px solid #ee1d25;
+    margin-bottom: 30px;
+}
+
+.title-solid h3 {
+    margin: 0;
+}
+
+.title-solid {
+    text-transform: uppercase;
+    background-color: #ee1d25;
+    padding: 10px 15px;
+    margin-bottom: 15px;
+    color: white;
+}
+
+.box-title {
+    font-size: large;
+    font-weight: bold;
+}
+
+.register-form .form-group {
+    margin-bottom: 25px;
+    position: relative;
+}
+.register-form label {
+    font-size: 14px;
+    font-weight: 400;
+}
```

Comment:
Da ne ponavljam gore komentar, isto tako vazi za ostalo.
Znaci jedan selektor za okruzujucu klasu, i onda u njoj gadjas prezicnije elemente.
Lakse ce ti biti da imas

```
.register-form {
label {
nesto...
}

.form-group {
nesto..
}
}
```
Sve ti je na jednom mestu tako, bolji pregled imas i lakse uocis gresku

Reply:
Jasno jasno, nisam znao za to uopste, vazi, ponavljace se to na vise mesta